proxied-token/proxied-token-spec.ini

```diff
@@ -0,0 +1,295 @@
+[root]
+k: #execute => #halt
+code: {PROXY_CODE}
+
+; This are our variables:
+; Default statusCode is EVMC_REVERT
+statusCode: _ => EVMC_REVERT
+output: _ => _
+callData:
+log: _
+refund: _ => _
+msg_sender: RANDOM_ID
+callValue: 0
+storage:
+    #hashedLocation({COMPILER}, {_MASTER_COPY}, .IntList) |-> MASTER_COPY_ID
+requires:
+
+; Proxy pattern functions
+
+[implementation]
+callData: #abiCallData("implementation", .TypedArgs)
+
+[implementation-success]
+statusCode: _ => EVMC_SUCCESS
+output: _ => #asByteStackInWidth(MASTER_COPY_ID, 32)
+
+[implementation-failure]
+; non-zero call value
+callValue: CALL_VALUE
++requires:
+    andBool #rangeUInt(256, CALL_VALUE)
+    andBool 0  <Int CALL_VALUE
```

Comment:
This sort of failure specs also look ambiguous. Please add status code specification for all.

Reply:
done